lib/bencher_json/src/project/measure/defs.rs

```diff
@@ -0,0 +1,128 @@
+use crate::JsonNewMeasure;
```

Comment:
I would prefer a more descriptive name for these definitions than `defs`. What are these definitions for?

My current preference would be for `built_in`. These are Measure definitions that are built-in to Bencher.
I'm open to suggestions though!

Reply:
The `defs` are of the parent module, `measure::defs` -> `Measure Definitions` ..
But happy to change to `built_in`. Will do!